src/Countries/Poland.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Poland extends Country
+{
+    public function countryCode(): string
+    {
+        return 'pl';
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Nowy Rok' => '01-01',
+            'Święto Trzech Króli' => '01-06',
+            'Święto Pracy' => '05-01',
+            'Święto Konstytucji 3 Maja' => '05-03',
+            'Święto Wojska Polskiego, Wniebowzięcie Najświętszej Maryi Panny' => '08-15',
+            'Wszystkich Świętych' => '11-01',
+            'Święto Niepodległości' => '11-11',
+            'Boże Narodzenie' => '12-25',
+            'Boże Narodzenie' => '12-26',
```

Comment:
Duplicate array key, this overwrites previous one resulting with only 12-26 being marked as holiday
```suggestion
            'Drugi Dzień Bożego Narodzenia' => '12-26',
```